README.md

```diff
@@ -142,11 +142,14 @@ app.on('start', () => {
 -------------------------------------------------------
 <a name="use"></a>
 
-### app.use(func, [opts])
+### app.use(func, [optsOrFunc])
 
 Loads one or more functions asynchronously.
+
 The function **must** have the signature: `instance, options, done`
 
+However, if the function is a `Promise` (i.e. `async`), it **must** have the signature: `instance, options`
```

Comment:
This should work also with `use(async function (instance) {})` or `use(async function () {})`. Why do you say it's required?

Reply:
You are correct, I didn't read into it but simply wanted to add a note for `async` functions as the first **must** does not hold in such functions. Fixed it in the latest commit.

Reply:
This does not seems fixed.

Also, the function is not a Promise, but _returns_ a promise.

Reply:
Sorry for wasting your time with minor issues, I fixed it in the latest commit.